src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/ApplyPreserveAttribute.cs

```diff
@@ -8,6 +8,10 @@
 
 using Mono.Cecil;
 
+#if NET5_LINKER
```

Comment:
Should we change this `#if` to `NETCOREAPP3_1` and we could remove this `$(DefineConstants)`?

https://github.com/xamarin/xamarin-android/blob/eadd3bda8e2a81c6ca997f05d3faac7040b29a7b/src/Microsoft.Android.Sdk.ILLink/Microsoft.Android.Sdk.ILLink.csproj#L5

If we change this `.csproj` to `TargetFramework=net5.0` later on, then `NETCOREAPP3_1` will still be defined.

Reply:
I think this was just changed recently (https://github.com/dotnet/sdk/pull/13615). So `NETCOREAPP3_1` will not work. I think we should use `#if NET` instead. I will try to use it.

Reply:
It is not defined yet in the version we are using, so I will keep `NET5_LINKER` around until we have the `NET` define.